.github/workflows/coder.yaml

```diff
@@ -30,9 +30,24 @@ concurrency:
   cancel-in-progress: ${{ github.event_name == 'pull_request' }}
 
 jobs:
+  changes:
```

Comment:
Could we use this instead? https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-excluding-paths

Reply:
oh yeah nice find